mdagent/tools/maketools.py

```diff
@@ -80,7 +79,7 @@ def make_all_tools(
         CheckDirectoryFiles(),
         #    InstructionSummary(path_registry=path_instance),
         ListRegistryPaths(path_registry=path_instance),
-        MapPath2Name(path_registry=path_instance),
+        #    MapPath2Name(path_registry=path_instance),
         Name2PDBTool(path_registry=path_instance),
```

Comment:
you sure we want to remove that tool? suppose there are some files provided by users - just a thought

Reply:
good point, is just that the agent sometimes adds the same file more than once with the tool taking it made it work better for my examples.